lib/api/devices.js

```diff
@@ -0,0 +1,22 @@
+export const registerDevice = ({ DeviceToken, DeviceName, DeviceModel, DeviceVersion, AppVersion, Environment, PublicKey }) => ({
+    method: 'post',
+    url: 'devices',
+    data: { DeviceToken, DeviceName, DeviceModel, DeviceVersion, AppVersion, Environment, PublicKey },
+});
+
+export const unregisterDevice = (UID, DeviceToken) => ({
```

Comment:
Why are you not using `UID`, `DeviceToken`?

Reply:
In https://gitlab.protontech.ch/ProtonMail/Slim-API/blob/develop/api-spec/pm_api_devices.md those two parameters don't appear neither in the body of the request (which would go into `data`) nor in the url (which would go into `params`), but they still appear as parameters in the api-spec. I thought it looked weird, but I "respected" what's written in the api-spec

Reply:
Remove `devises.js` file please, it's just use by mobile team.